actors/cloudbroker/cloudbroker__computenode/methodclass/cloudbroker_computenode.py

```diff
@@ -90,7 +89,7 @@ def sync(self, locationId, **kwargs):
     @auth(['level2', 'level3'], True)
     def enableStacks(self, ids, **kwargs):
         kwargs['ctx'].events.runAsync(self._enableStacks,
-                                      args=(ids, ),
+                                      args=(ids),
```

Comment:
Are you sure this is correct. It used to be a tuple, and like this the () does not do anything